ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -72,8 +72,8 @@ export const getServerSideProps: GetServerSideProps = async context => {
     props: {
       ...baseProps,
       pageType: HOME_PAGE,
-      service: 'pidgin',
-      pathname: '/pidgin',
+      service: 'ws',
+      pathname: '/ws/languages',
```

Comment:
I've just noticed in the fixture data, the `pageIdentifier` is `ws.page` whereas on the current live page - https://www.bbc.com/ws/languages - we send a value of p = `ws.languages.page` to Piano (page view tracker). 

I think we need to manipulate the `pageIdentifier` here, to ensure it's correct - we need to override the `metadata.atiAnalytics` object & set `pageIdentifier` to `ws.languages.page`.

We should also add a scenario to the cypress/specialFeatures/atiAnalytics for the /ws/languages page if it doesn't already exist